lib/sles4sap/console_redirection.pm

```diff
@@ -218,7 +203,7 @@ sub connect_target_to_serial {
 
     if (check_serial_redirection()) {
         record_info('Redirect ON', "Console is already redirected to:" . script_output('hostname', quiet => 1));
-        return;
+        return 1;
```

Comment:
UT did not like if nothing was returned.

Reply:
Could you please elaborate?

Reply:
Hmm... so the expected behavior is that if redirection is already active , function exists entering if statement at L110 and must not reach the L124. 
In the UT I changed `enter_cmd` to just die: ```$redirect->redefine(enter_cmd => sub { die; });```
The UT just tests if function exits without issue, but with `redefine` above this happens only after entering if statement and return at L112. If it reaches L124, function is forced to fail.